NOTICE.md

```diff
@@ -15,7 +15,7 @@ The core module includes the following software:
  * Apache Commons Collections - we copied parts of the BinaryHeap (Apache License)
  * java-string-similarity - we copied the implementation of JaroWinkler (MIT license)
  * com.fasterxml.jackson.core:jackson-annotations (Apache License)
- * com.vividsolutions:jts (LGPL), see #1039
+ * org.locationtech:jts (LGPL), see #1039
```

Comment:
Due to the new community we can now use a different license. I would choose the EDL (BSD alike) as the EPL is closer to GPL e.g. one has to release source code: https://tldrlegal.com/license/eclipse-public-license-1.0-(epl-1.0)